# CancelSpeech.srv
---
bool success
string message
